F/flex/build_tarballs.jl

```diff
@@ -0,0 +1,38 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "flex"
+version = v"2.6.4"
+
+# Collection of sources required to complete build
+sources = [
+    ArchiveSource("https://github.com/westes/flex/releases/download/v2.6.4/flex-2.6.4.tar.gz", "e87aae032bf07c26f85ac0ed3250998c37621d95f8bd748b31f15b33c45ee995")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+apk add gettext-dev
```

Comment:
Would having `Gettext_jll` as `HostBuildDependency` work?

Reply:
Yes

Reply:
So can remove this one?

Reply:
Why does this have to stay?  What's the problem with `Gettext_jll`?

Reply:
My change wasn't accounted for when I tested for some reason, testing later it doesn't build with the jll. I will work on fixing that tonight.